app/views/scheduled_pickups/_scheduled_pickup.html.erb

```diff
@@ -3,6 +3,6 @@
     <%= t(".header") %>
   </dt>
   <dd>
-    <%= format_pickup_time(scheduled_pickup) %>
+    <%= scheduled_pickup.time_range %>
```

Comment:
Will this print out the correct format or just output the object? Do you have to write it like this for `to_s` to be called on it:

```ruby
<%= #{scheduled_pickup.time_range} %>
```